parser/expression.go

```diff
@@ -196,11 +200,18 @@ func (self *_parser) parseVariableDeclarationList(var_ file.Idx) []ast.Expressio
 	var list []ast.Expression
 
 	for {
-		list = append(list, self.parseVariableDeclaration(&declarationList))
+		comments := self.findComments(false)
+
+		decl := self.parseVariableDeclaration(&declarationList)
+		if self.mode & StoreComments != 0 { self.commentMap.AddComments(decl, comments, ast.LEADING) }
```

Comment:
Don't test twice and avoid trailing blocks.

Have you run gofmt on this as I didn't think it allowed trailing blocks?